tools/usfm-markers/src/generators/markers-data/utils/categoriesMap.ts

```diff
@@ -0,0 +1,304 @@
+  thc8: "CenterTables",
+  thc9: "CenterTables",
+  thc10: "CenterTables",
+  thc11: "CenterTables",
+  thc12: "CenterTables",
+  tcc1: "CenterTables",
+  tcc2: "CenterTables",
+  tcc3: "CenterTables",
+  tcc4: "CenterTables",
+  tcc5: "CenterTables",
+  tcc6: "CenterTables",
+  tcc7: "CenterTables",
+  tcc8: "CenterTables",
+  tcc9: "CenterTables",
+  tcc10: "CenterTables",
+  tcc11: "CenterTables",
+  tcc12: "CenterTables",
+  thr1: "RightTables",
+  thr2: "RightTables",
+  thr3: "RightTables",
+  thr4: "RightTables",
+  thr5: "RightTables",
+  thr6: "RightTables",
+  thr7: "RightTables",
+  thr8: "RightTables",
+  thr9: "RightTables",
+  thr10: "RightTables",
+  thr11: "RightTables",
+  thr12: "RightTables",
+  tcr1: "RightTables",
+  tcr2: "RightTables",
+  tcr3: "RightTables",
+  tcr4: "RightTables",
+  tcr5: "RightTables",
+  tcr6: "RightTables",
+  tcr7: "RightTables",
+  tcr8: "RightTables",
+  tcr9: "RightTables",
+  tcr10: "RightTables",
+  tcr11: "RightTables",
+  tcr12: "RightTables",
+  lh: "Lists",
+  li: "Lists",
+  li1: "Lists",
+  li2: "Lists",
+  li3: "Lists",
+  li4: "Lists",
+  lf: "Lists",
+  lim: "Lists",
+  lim1: "Lists",
+  lim2: "Lists",
+  lim3: "Lists",
+  lim4: "Lists",
+  litl: "Lists",
+  lik: "Lists",
+  liv: "Lists",
+  liv1: "Lists",
+  liv2: "Lists",
+  liv3: "Lists",
+  liv4: "Lists",
+  liv5: "Lists",
+  f: "Footnotes",
+  fe: "Footnotes",
+  fr: "Footnotes",
+  ft: "Footnotes",
+  fk: "Footnotes",
+  fq: "Footnotes",
+  fqa: "Footnotes",
+  fl: "Footnotes",
+  fw: "Footnotes",
+  fp: "Footnotes",
+  fv: "Footnotes",
+  fdc: "Footnotes",
+  fm: "Footnotes",
+  x: "CrossReferences",
+  xo: "CrossReferences",
+  xop: "CrossReferences",
+  xt: "CrossReferences",
+  xta: "CrossReferences",
+  xk: "CrossReferences",
+  xq: "CrossReferences",
+  xot: "CrossReferences",
+  xnt: "CrossReferences",
+  xdc: "CrossReferences",
+  rq: "CrossReferences",
+  qt: "SpecialText",
+  nd: "SpecialText",
+  tl: "SpecialText",
+  dc: "SpecialText",
+  bk: "SpecialText",
+  sig: "SpecialText",
+  pn: "SpecialText",
+  png: "SpecialText",
+  addpn: "SpecialText",
+  wj: "SpecialText",
+  k: "SpecialText",
+  sls: "SpecialText",
+  ord: "SpecialText",
+  add: "SpecialText",
+  lit: "SpecialText",
+  no: "CharacterStyling",
+  it: "CharacterStyling",
+  bd: "CharacterStyling",
+  bdit: "CharacterStyling",
+  em: "CharacterStyling",
+  sc: "CharacterStyling",
+  sup: "CharacterStyling",
+  pb: "Breaks",
+  fig: "SpecialFeatures",
+  jmp: "SpecialFeatures",
+  pro: "SpecialFeatures",
+  rb: "SpecialFeatures",
+  w: "PeripheralReferences",
+  wh: "PeripheralReferences",
+  wg: "PeripheralReferences",
+  wa: "PeripheralReferences",
+  ndx: "PeripheralReferences",
+  periph: "PeripheralMaterials",
+  p1: "PeripheralMaterials",
+  p2: "PeripheralMaterials",
+  k1: "PeripheralMaterials",
+  k2: "PeripheralMaterials",
+  xtSee: "PeripheralMaterials",
+  xtSeeAlso: "PeripheralMaterials",
+} as { [marker: string]: CategoryType };
```

Comment:
Using `as` (type assertion) overrides typing in TS and should be typically avoided. Here if you added `unknown: "Unknown",` to the object it wouldn't complain. However if you change to:
```ts
const categoriesMap: { [marker: string]: CategoryType } = {
  ...
};
export default categoriesMap;
```
and then added `unknown: "Unknown",` to the object it would complain appropriately.

Reply:
Yeah I know, it is a big change for something that is not going to change very often, because it means changing each value to be a string of CategoryType and not just a string, otherwise it complains. I'll change it.